src/main/java/momo/command/AddCommand.java

```diff
@@ -0,0 +1,43 @@
+package momo.command;
+
+import momo.Storage;
+import momo.StorageException;
+import momo.task.Task;
+
+import java.io.IOException;
+
+import static momo.Momo.FILE_PATH;
+
+/**
+ * The {@code AddCommand} class provides implementations for adding tasks
+ * to the storage and for printing confirmation messages that display tasks
+ * which have been added
+ */
+public class AddCommand extends Command {
```

Comment:
Consider using a descriptive name. The name "AddCommand" could be mistaken for mathematical addition. Maybe consider using "AddTaskCommand"?

Reply:
Good point! I also realised it was an issue because the coding quality section addresses how class names should be nouns not verbs. So I think I might change it to "CommandAdder" instead